snagsby/sources.py

```diff
@@ -96,9 +136,13 @@ def _parse_sources_str(sources_str):
         for source in SPLITTER.split(sources_str) if source
     ]
 
+def parse_source(source):
```

Comment:
This is nice. Let's write some unit tests around this method for the three cases of s3, sm, or invalid.